microhapulator/data/template.html

```diff
@@ -21,6 +21,12 @@
             img.small {
                 width: 24.5%;
             }
+            .centerfull {
+            width: 100%;
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+            }           
```

Comment:
Also, to be clear, my suggestion was to apply two separate classes to these image tags: "center" (which you had just created I think) and "full", which had not yet been created. Often times it's helpful to define granular style classes that can be re-used across different tags and tag types, and you can apply multiple classes to a tag by separating their class names with a space in the tag `class=""` attribute.

But of course, that's all moot now and we can probably drop the `centerfull` class. :smiley: